client.js

```diff
@@ -53,21 +51,36 @@ module.exports = function (opts, cb) {
   ])
 
   ms.client(remote, function (err, stream) {
-    if(err) return cb(explain(err, 'could not connect to sbot'))
-    var id = '@'+stream.remote.toString('base64')+'.ed25519'
-    // mix: id === config.keys.id, note sure why this is needed
-    var sbot = muxrpc(manifest, false, null, id)
+    if(err) {
+      return cb(explain(err, 'could not connect to sbot'))
+    }
 
-    // fix blobs.add. (see ./util/fix-add-blob.js)
-    if (sbot.blobs && sbot.blobs.add)
-      sbot.blobs.add = fixBlobsAdd(sbot.blobs.add)
+    // HACK: This refers to the *remote* feed ID, not our local ID, and we don't
+    // actually have a way of determining the feed type. This is fine when all
+    // feeds are ed25519, but will cause problems in the future.
+    const remoteId = `@${stream.remote.toString('base64')}.ed25519`
```

Comment:
Would love another pair of eyes to ensure that this looks right.

Reply:
good comment!